tests/jenkins/install_neuron_reportinglib.sh

```diff
@@ -21,7 +21,13 @@ patch_neuron() (
 set -e
 source ${JENKINS_DIR:-.}/_env_setup.sh
 
+# Install reportinglib with spack to run reportinglib tests
+spack install reportinglib%intel &
+pid=$!
+
```

Comment:
I am not a big fan of this because we miss information about the installation of `reportinglib` while we gain only ~20 seconds.